main.py

```diff
@@ -7,6 +7,11 @@ class Preprocess(Worker):
     def __init__(self):
         self.processor = WhisperProcessor.from_pretrained("openai/whisper-base")
 
+    def deserialize(self, data: bytes) -> any:
+        with io.BytesIO(data) as byte_io:
+            array,sampling_rate =soundfile.read(byte_io)
+        return {'array':array,'sampling_rate':sampling_rate}
```

Comment:
Since the request body is a binary, I think the response should also change to raw instead of JSON.

Reply:
You mean the return of func *deserialize* should be raw? Is the code below ok?
```python
    def deserialize(self, data: bytes) -> any:
        with io.BytesIO(data) as byte_io:
            return soundfile.read(byte_io)

    def forward(self, data):
        array,sampling_rate=data
        res = self.processor(array, sampling_rate=sampling_rate, return_tensors="pt")
        return res.input_features
```

Reply:
I mean the serialisation of egress.